shoop/admin/modules/orders/dashboard.py

```diff
@@ -45,25 +58,31 @@ def get_subtitle(count):
     return _("Based on %d orders") % count
 
 
-def get_sales_of_the_day_block(request):
+def get_sales_of_the_day_block(currency, request):
```

Comment:
`request` is usually the first argument, I'd appreciate it if it was the same here.